fury/actors/peak.py

```diff
@@ -0,0 +1,343 @@
+
+        points_array = np.asarray(points_array)
+        centers_array = np.asarray(centers_array)
+        diffs_array = np.asarray(diffs_array)
+
+        vtk_points = numpy_to_vtk_points(points_array)
+
+        vtk_cells = PeakActor._points_to_vtk_cells(points_array)
+
+        peaks_colors_object = PeakActor._peaks_colors_from_points(
+            points_array, colors=colors)
+        vtk_colors, colors_are_scalars, self.__go = peaks_colors_object
+
+        poly_data = vtk.vtkPolyData()
+        poly_data.SetPoints(vtk_points)
+        poly_data.SetLines(vtk_cells)
+        poly_data.GetPointData().SetScalars(vtk_colors)
+
+        self.__mapper = vtk.vtkPolyDataMapper()
+        self.__mapper.SetInputData(poly_data)
+        self.__mapper.ScalarVisibilityOn()
+        self.__mapper.SetScalarModeToUsePointFieldData()
+        self.__mapper.SelectColorArray('colors')
+        self.__mapper.Update()
+
+        self.SetMapper(self.__mapper)
+
+        attribute_to_actor(self, centers_array, 'center')
+        attribute_to_actor(self, diffs_array, 'diff')
+
+        vs_dec_code = load('peak_dec.vert')
+        vs_impl_code = load('peak_impl.vert')
+        fs_dec_code = load('peak_dec.frag')
+        fs_impl_code = load('peak_impl.frag')
+
+        shader_to_actor(self, 'vertex', decl_code=vs_dec_code,
+                        impl_code=vs_impl_code)
+        shader_to_actor(self, 'fragment', decl_code=fs_dec_code)
+        shader_to_actor(self, 'fragment', impl_code=fs_impl_code,
+                        block='light')
+
+        # Color scale with a lookup table
+        if colors_are_scalars:
+            if lookup_colormap is None:
+                lookup_colormap = colormap_lookup_table()
+
+            self.__mapper.SetLookupTable(lookup_colormap)
+            self.__mapper.UseLookupTableScalarRangeOn()
+            self.__mapper.Update()
+
+        self.__lw = linewidth
+        self.GetProperty().SetLineWidth(self.__lw)
+
+        if self.__go >= 0:
+            self.GetProperty().SetOpacity(self.__go)
+
+        self.__min_centers = np.min(indices, axis=1)
+        self.__max_centers = np.max(indices, axis=1)
+
+        self.__is_range = True
+        self.__low_ranges = self.__min_centers
+        self.__high_ranges = self.__max_centers
+        self.__cross_section = self.__high_ranges // 2
+
+        self.__mapper.AddObserver(vtk.vtkCommand.UpdateShaderEvent,
+                                  self.__display_peaks_vtk_callback)
+
+    @staticmethod
+    def _orientation_colors(points, cmap='rgb_standard'):
+        """
+
+        Parameters
+        ----------
+        points : (N, 3) array or ndarray
+            points coordinates array.
+        cmap : string ('rgb_standard', 'boys_standard')
+            colormap.
+
+        Returns
+        -------
+        colors_list : ndarray
+            list of  Kx3 colors. Where K is the number of lines.
+
+        """
+        if cmap == 'rgb_standard':
```

Comment:
you should add before `cmap = cmap.lower()`

Reply:
Changed. Thanks for the tip.